lib/Dialect/SV/Transforms/SVExtractTestCode.cpp

```diff
@@ -461,7 +477,8 @@ void SVExtractTestCodeImplPass::runOnOperation() {
 
   auto &instanceGraph = getAnalysis<circt::hw::InstanceGraph>();
 
-  for (auto &op : topLevelModule->getOperations()) {
+  BindTable bindTable;
+  for (auto &op : llvm::make_early_inc_range(topLevelModule->getOperations())) {
     if (auto rtlmod = dyn_cast<hw::HWModuleOp>(op)) {
```

Comment:
Nit: not totally related, but this could just do a `topLevelModule->getOps<hw::HWModuleOp>()` and save this whole `if` nesting.

Reply:
This whole pass can use a clean up. Besides this, there are also places where we create a builder, and rather than moving the insertion point, create a whole new builder a couple lines later. I think a general ETC cleanup PR is probably in order, but for now I'm trying to make the least changes.